### PR TITLE
UI setup

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
Created Video library all Ui components.

Preview Image :
fabtube

### CODE REVIEW

File: src/components/cards/VideoCard.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
```
Comment (devsurajrai) [inline src/components/cards/VideoCard.js:1]: we don't need it in latest version of react

File: src/components/cards/videoCard.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.video-card {
```
Comment (devsurajrai) [inline src/components/cards/videoCard.css:2]: you can use comments for the styles here.

File: src/components/header/Header.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { Menu } from "@mui/icons-material";
+import NotificationsNoneOutlinedIcon from "@mui/icons-material/NotificationsNoneOutlined";
+import AccountCircleSharpIcon from "@mui/icons-material/AccountCircleSharp";
+import SearchIcon from "@mui/icons-material/Search";
```
Comment (devsurajrai) [inline src/components/header/Header.js]: you can have a separate file for mui icons and export from there, in that way you will be importing your icons only from a single source.

**Review by devsurajrai — COMMENTED** [review]
Everything looks good. Good Work.
* Try to implement the suggestion if you find them useful

File: src/components/chips/chips.css [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-padding: 0.8rem 4rem;
-background-color: var(--primary);
-border-bottom: 1px solid var(--primary-inverse-20);
-border-top: 1px solid var(--primary-inverse-20);
-position: relative;
+.chips-bar {
+  width: 83%;
+  height: 3.8rem;
+  padding: 0.8rem 4rem;
+  background-color: var(--primary);
```
Comment (Shubhamkr2610) [inline src/components/chips/chips.css]: use px for less than 1rem value

File: src/components/sidebar/sidebar.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  height: 90vh;
+}
+.sidebar-section {
+  border-bottom: 1px solid var(--primary-inverse-20);
+  display: flex;
+  flex-direction: column;
+  padding:  1rem 0;
+}
+.sidebar-bar {
+    padding: 0.5rem 1.5rem;
```
Comment (Shubhamkr2610) [inline src/components/sidebar/sidebar.css:17]: use px value  for less than 1rem

**Review by Shubhamkr2610 — APPROVED** [review]
code looks clean , easy to understand

File: src/components/cards/videoCard.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+.bottom {
+  height: 50%;
+  width: 100%;
+  padding: 0.5rem;
+  display: flex;
+}
+.bottom .title-wrapper {
+  width: 85%;
+  margin-left: 0.5rem;
+}
```
Comment (Shubhamkr2610) [inline src/components/cards/videoCard.css:27]: use px values for less than 1rem

**Review by Shubhamkr2610 — APPROVED** [review]
I can see that you are using rem values for less than 1rem , it's bad practice .Please change them to px values .  Else good naming with classes  . UI is awesome . Code is clean  and readable , You are good to go .

**Pull request merged**